docs/plugins/survey.md

```diff
@@ -2,7 +2,7 @@
 
 Current version: 2.0.0. [See version history](https://github.com/jspsych/jsPsych/blob/main/packages/plugin-survey/CHANGELOG.md).
```

Comment:
So the package version numbers in _unpkg links_ will update automatically when the docs are built/released, but not anywhere else, right? If so, I need to edit the plugin version number here.

Reply:
This should also update automatically

Reply:
Oops, just realized that the docs changes aren't live so they just need to be pushed, right?

Reply:
Awesome. Sorry that I forgot how to do this and thanks for letting me know!